### PR TITLE
reformat agentQnA sample guide

### PR DESCRIPTION
**Pull request by abdulari**
this PR relates to #150

### CODE REVIEW

**Comment by chickenrae** [comment]
@abdulari Are you taking on issue #150 or simply reformating?

File: examples/AgentQnA/AgentQnA_Guide.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.. note:: This guide is in its early development and is a work-in-progress with
+   placeholder content.
+
+Overview
+********
+
+This example showcases a hierarchical multi-agent system for question-answering applications. 
+
+Purpose
+*******
```
Comment (xuechendi) [inline examples/AgentQnA/AgentQnA_Guide.rst:15]: @minmin-intel, please check the purpose content below.

**Comment by xuechendi** [comment]
is there a reason we only provide xeon doc, how about Gaudi?

**Comment by minmin-intel** [comment]
@abdulari We need to provide guides for deployment on Gaudi to use open-source LLMs. Currently, deployment on Xeon requires using OpenAI GPT-4o model API.

**Review by tomlenth — APPROVED** [review]
approving although unclear if there is a link failure

**Review by yinghu5 — APPROVED** [review]
merged it and then update later

**Pull request merged**